### DISCUSSION TITLE
Welcome to circumplex Discussions!

### DISCUSSION THREAD

**Discussion by jmgirard** [jmgirard/circumplex#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Marco3231st** [comment]
Hi,
I am new to R projets. I am trying to work with your circumplex project, but unfortunatly I cant get the devtool package to work.
![image](https://github.com/jmgirard/circumplex/assets/137517291/ba33946f-db85-4542-bd6e-614a15402ae3)
It seems like I downloaded it right but as you can see in the next picture, the > devtools::install_github("jmgirard/circumplex") comande doesn't want to work...
![image](https://github.com/jmgirard/circumplex/assets/137517291/ee9f5776-b996-42ad-b660-81d589a0cfc6)
It's probably a rookie mistake, but do you have an idea why it doesn't work? 

Thanks!

**Reply by jmgirard** [reply]
That isn't an error. It is just telling you that it isn't going to install the package because you already have that version of the package installed. I recommend looking up an introduction to R book, video, or training.
